evm_arithmetization/src/memory/columns.rs

```diff
@@ -42,8 +42,25 @@ pub(crate) const VIRTUAL_FIRST_CHANGE: usize = SEGMENT_FIRST_CHANGE + 1;
 // Contains `next_segment * addr_changed * next_is_read`.
 pub(crate) const INITIALIZE_AUX: usize = VIRTUAL_FIRST_CHANGE + 1;
 
+// Contains the list of pruned contexts. Each pruned context must appear exactly
```

Comment:
It sounds odd that a single cell contains many values (I know it's because you're talking about the `PRUNED_CONTEXTS` cells of all rows). What about this?
```suggestion
// Contains the `row_index` if and only if context `row_index` must be pruned```